example/src/App.tsx

```
@@ -1,441 +1,751 @@
+          logAndUpdateState('Error: ' + (e as Error).message);
+        }
+      },
+    },
+    {
+      label: 'Send Batches',
+      action: async (logAndUpdateState: (msg: string) => void) => {
+        try {
+          const result = await kit.sendBatches();
+          let msg = 'Send batches: ' + JSON.stringify(result);
+          // Check which batches were removed
+          if (result && result.batches) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)